### PR TITLE
task/frontend40 + sidebar studio component

### PR DESCRIPTION
**Pull request by so5tmaker**

### CODE REVIEW

File: frontend/src/components/auth/components/studio/studio.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  const [buttonName, setName] = useState<string>(IconName.HOME);
+
+  const handleClick = (name: string): void => {
+    setName(name);
+  };
+
+  return (
+    <div className={styles.studio}>
+      <StudioSidebar onClick={handleClick} />
+      {buttonName === IconName.HOME ? <StudioHome /> : <StudioAnalystics />}
```
Comment (VitaliiBedletskyi) [inline frontend/src/components/auth/components/studio/studio.tsx:21]: It seems better to use a router for switching pages

File: frontend/src/components/auth/auth.tsx [hunk 2]
```diff
@@ -48,6 +48,9 @@ const Auth: FC = () => {
           />
         );
       }
+      case AppRoute.STUDIO: {
```
Comment (VitaliiBedletskyi) [inline frontend/src/components/auth/auth.tsx]: It shouldn't be in the authorization component

File: frontend/src/components/app/app.tsx [hunk 3]
```diff
@@ -1,19 +1,45 @@
...
   return (
     <>
-      <Routes>
-        <Route path={AppRoute.ROOT} element="Root" />
-        <Route path={AppRoute.SIGN_UP} element={<Auth />} />
-        <Route path={AppRoute.SIGN_IN} element={<Auth />} />
-        <Route path={AppRoute.RESTORE_PASSWORD} element={<Auth />} />
-        <Route path={AppRoute.ANY} element={<NotFound />} />
-      </Routes>
+      <img src={logo} className="App-logo" width="30" alt="logo" />
```
Comment (VitaliiBedletskyi) [inline frontend/src/components/app/app.tsx]: Get rid of these blocks of code:
````
<img src={logo} className="App-logo" width="30" alt="logo" />
````
````
<ul className="App-navigation-list">
        <li>
          <Link to={AppRoute.ROOT}>Root</Link>
        </li>
        <li>
          <Link to={AppRoute.SIGN_IN}>Sign in</Link>
        </li>
        <li>
          <Link to={AppRoute.SIGN_UP}>Sign up</Link>
        </li>
        <li>
          <Link to={AppRoute.STUDIO}>Studio</Link>
        </li>
      </ul>
      <p>Current path: {pathname}</p>
````

**Review by VitaliiBedletskyi — CHANGES_REQUESTED** [review]

File: frontend/src/components/auth/auth.tsx (hunk 2 above)
Comment (so5tmaker) [reply frontend/src/components/auth/auth.tsx]: Done!

**Pull request closed without merging**